site/search/SearchPage.tsx

```diff
@@ -20,12 +21,29 @@ export const SearchPage = (props: {
     return (
         <Html>
             <Head
-                canonicalUrl={`${baseUrl}/data`}
-                pageTitle="Data Catalog"
-                pageDesc="Explore Our World in Data's extensive collection of charts. Use the search bar to find specific data visualizations or browse by topic. Filter by country or subject area to discover insights on global issues supported by reliable data."
+                canonicalUrl={`${baseUrl}${SEARCH_BASE_PATH}`}
+                pageTitle="Search"
+                pageDesc="Search articles and charts on Our World in Data. Filter by country or subject area to discover insights on global issues supported by reliable data."
                 baseUrl={baseUrl}
                 imageUrl={`${baseUrl}/data-catalog-thumbnail.png`}
             >
+                <script
+                    type="application/ld+json"
+                    dangerouslySetInnerHTML={{
+                        // Structured data for google
+                        __html: JSON.stringify({
+                            "@context": "https://schema.org",
+                            "@type": "WebSite",
+                            url: baseUrl,
+                            potentialAction: {
+                                "@type": "SearchAction",
+                                target: `${baseUrl}${SEARCH_BASE_PATH}?q={search_term_string}`,
+                                "query-input":
+                                    "required name=search_term_string",
+                            },
+                        }),
+                    }}
+                />
```

Comment:
Why are we adding this? Google [no longer supports](https://developers.google.com/search/blog/2024/10/sitelinks-search-box) the feature this enabled.

Reply:
Good to know, removed in https://github.com/owid/owid-grapher/pull/5397